pkg/systemd/logDetails.jsx

```diff
@@ -55,7 +57,7 @@ const LogDetails = ({ entry }) => {
                     </CardHeaderMain>
                     { service &&
                         <CardActions>
-                            <Button variant="link" onClick={() => cockpit.jump("/system/services#/" + service) }>
+                            <Button variant="link" onClick={() => cockpit.jump("/system/services#/" + service + (is_user ? "?owner=user" : "")) }>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18032-20230110-142642-9203b7d7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)